README.md

```diff
@@ -86,123 +86,6 @@ $ tw info
      Authentication API credential's token | OK
 ```
 
-### 4. View available commands
-
-Use the `-h` or `--help` parameter to list the available commands and their associated options.
-
-```bash
-tw --help
-```
-
-### 5. Add credentials
-
-To launch pipelines on AWS Batch, you will need to add credentials to the appropriate Tower Workspace.
-
-```bash
-tw credentials add aws --name=my_aws_creds --access-key=<aws access key> --secret-key=<aws secret key>
-```
-
-> See the [IAM policy](https://github.com/seqeralabs/nf-tower-aws/tree/master/forge) for Tower Forge for recommendations on AWS Batch permissions.
-
-### 6. List credentials
-
-List the credentials available in the workspace.
-
-```bash
-tw credentials list
-```
-
-### 7. Provision a Compute Environment
-
-Create a Compute Environment for AWS Batch with automatic provisioning of cloud computing resources:
-
-```bash
-tw compute-envs add aws-batch forge --name=my_aws_ce --region=eu-west-1 --max-cpus=256 --work-dir=s3://<bucket name> --wait=AVAILABLE
-```
-
-The above command will create all of the required AWS Batch resources in the AWS Ireland (`eu-west-1`) region with a total of 256 CPUs provisioned in the compute environment. An existing S3 bucket will be used as the work directory when running Nextflow. Also, it will wait until all the resources are AVAILABLE and ready to use.
```

Comment:
"with a total of 256 CPUs" maybe is better to say "with a maximum of 256 CPUs" to stress that they are not provisioned at creation time, only at run time if needed.

Reply:
Ah, good catch - fixing it now. Thanks mate 👍

Reply:
Addressed with https://github.com/seqeralabs/tower-cli/pull/231/commits/ab9690fb77e9d2ca3bf8d3c439bc4d038ce7e873